icpc-wf/ansible/roles/grafana/tasks/main.yml

```diff
@@ -0,0 +1,61 @@
+---
+# These tasks install and configure grafana
+
+- name: install domserver required packages
+  apt:
+    state: present
+    pkg:
+      - prometheus-nginx-exporter
```

Comment:
@eldering I assume you're aware but to make it explicit,

prometheus-nginx-exporter should be running on the machine running nginx, so this task should run together with the setup of the domserver.

prometheus-node-exporter should run with all hosts we care for (so at minimum the domserver+judgehosts) I see no reason to also monitor the admin machines.

Reply:
Yeah I think we need to install this always when some var is set, on the judgehosts and domserver

Reply:
Ok, I think there should then be a `prometheus_server` and `prometheus_client` role, and the client role might need some variables to know whether it should install `prometheus-nginx-exporter`. Or that could be put in a separate role again.